submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/ExtSubmissionRepository.kt

```diff
@@ -19,34 +19,33 @@ class ExtSubmissionRepository(
     private val toExtSubmissionMapper: ToExtSubmissionMapper,
     private val toDocSubmissionMapper: ToDocSubmissionMapper
 ) {
-    fun saveSubmission(submission: ExtSubmission, draftKey: String?): ExtSubmission {
-        updateCurrentRecords(submission.accNo, submission.owner, submission.submitter, draftKey)
-        return toExtSubmissionMapper.toExtSubmission(saveSubmission(submission), false)
+    fun saveSubmission(submission: ExtSubmission): ExtSubmission {
+        subDataRepository.expireActiveProcessedVersions(submission.accNo)
```

Comment:
lets include TODO to we remeber to  remove this side effect from here too.

Reply:
I moved it to a separate method